### DISCUSSION TITLE
GitHub projects shows empty issue card

### DISCUSSION THREAD

**Discussion by WasiqB** [community/community#48242] [Projects and Issues]
### Select Topic Area

Bug

## Empty issue card in Project

Since few days I have noticed that when I click on any ticket from the GitHub Projects, it opens an Empty ticket even when the issue is already created and visible in `Issues` tab in the repository.

Screenshot for the same is as below:

![Screenshot 2023-02-22 at 11 40 57 AM](https://user-images.githubusercontent.com/9130909/220538021-fc6e1296-4308-4a1f-96f1-f08a6fb99ec5.png)

**Comment by SMR39** [comment]
@rileybroughten ^^^ adding for visibility

**Comment by rileybroughten** [comment]
@WasiqB could you provide more details on your setup? Is this only happening for some of the issues in your project, or all? Is this a private project / repository or public where we could try and reproduce on our end?

**Comment by anyscalesam** [comment]
i have the same issue - private repo; although was able to repro just now on a private project in public repo https://github.com/ray-project/ray/

can we get an update on this; this was working for me just yesterday @rileybroughten

**Comment by ataber11** [comment]
I'm also having this issue. If I click "open in new tab" though, all of the info appears. It is happening for all of the issues in my project, and I didn't make any changes to my settings. Other teammates that are working from the same project, are not having these display issues either.

**Comment by ataber11** [comment]
FIXED! tested in another browser (safari) and it worked there, so assumed it had something to do with the how Chrome was displaying it. Quite chrome and relaunched (without automatic reopening of tabs) and everything is working fine.
